### PR TITLE
[Feature]: Add pretraining for FGIA

### PR DESCRIPTION
**Pull request by YuanLiuuuuuu**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here and update the documentation.

## Checklist

**Before PR**:

- [ ] Pre-commit or other linting tools are used to fix the potential lint issues.
- [ ] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects, like MMDet or MMSeg.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: projects/fgia_accv2022_1st/config/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# This is the official implementation of ViT-L pre-training for the 1st place of FGIA ACCV2022
+
```
Comment (fangyixiao18) [inline projects/fgia_accv2022_1st/config/README.md]: Better to indicate the version for pretraining

File: projects/fgia_accv2022_1st/config/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+# node 1
+NNODES=2 NODE_RANK=0 PORT=${MASTER_PORT} MASTER_ADDR=${MASTER_ADDR} bash tools/dist_train.sh projects/fgia_accv2022_1st/config/mae_vit-large-p16_8xb512-amp-coslr-1600e_in1k.py 8
+```
+
+```shell
+# node 2
+NNODES=2 NODE_RANK=1 PORT=${MASTER_PORT} MASTER_ADDR=${MASTER_ADDR} bash tools/dist_train.sh projects/fgia_accv2022_1st/config/mae_vit-large-p16_8xb512-amp-coslr-1600e_in1k.py 8
+```
+
+All these logs and checkpoints will be saved under the folder `work_dirs`in the root.
```
Comment (fangyixiao18) [inline projects/fgia_accv2022_1st/config/README.md]: add link of downstream task in mmcls

File: projects/fgia_accv2022_1st/config/README.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+# This is the official implementation of ViT-L pre-training for the 1st place of FGIA ACCV2022
```
Comment (tonysy) [inline projects/fgia_accv2022_1st/config/README.md]:
```suggestion
# Solution of FGIA ACCV 2022(1st Place )

This is the official implementation of ViT-L pre-training for the 1st place of FGIA ACCV2022
```

File: projects/fgia_accv2022_1st/config/README.md (hunk 1 above)
Comment (tonysy) [reply projects/fgia_accv2022_1st/config/README.md]: We can use the collect_env to list important information, like, mmcv, mmengine, mmcls, and pytorch, cuda, torchvision

**Comment by fangyixiao18** [comment]
please move README.md to `projects/fgia_accv2022_1st/` directory, currently it's in `projects/fgia_accv2022_1st/configs`

**Review by tonysy — APPROVED** [review]
LGTM

**Review by fangyixiao18 — APPROVED** [review]

**Pull request merged**